<?php

namespace ContainerEVVjX0P;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\ContainerInterface;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getErrorControllershowErrorPageService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.aMhho.e.App\Controller\ErrorController::showErrorPage()' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.aMhho.e.App\\Controller\\ErrorController::showErrorPage()'] = (new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService ??= $container->getService(...), [
            'exception' => ['privates', '.errored..service_locator.aMhho.e.Throwable', NULL, 'Cannot autowire service ".service_locator.aMhho.e": it references interface "Throwable" but no such service exists. Did you create an instantiable class that implements this interface?'],
            'logger' => ['privates', 'logger', 'getLoggerService', false],
        ], [
            'exception' => 'Throwable',
            'logger' => '?',
        ]))->withContext('App\\Controller\\ErrorController::showErrorPage()', $container);
    }
}
